src/components/Essentials/essentialsfiltersmobile.js

```diff
@@ -25,6 +25,7 @@ const useInputLabelStyles = makeStyles(() => ({
   root: {
     fontFamily: 'archia',
     fontSize: '11px !important',
+    minHeight: '14px',
```

Comment:
This should be added in the object `useMenuItemStyles` just like in the `essentialsfiltersdesktop.js` file.
Also after doing additional testing in 4k display, it seems we require minHeight: '15px' 
Please do this for both desktop and mobile.